app/views/certificates.html

```diff
@@ -0,0 +1,209 @@
+<div class="page-content">
+  <div class="row">
+    <!-- include the sidebar -->
+    <div ng-include="'views/sidebar.html'"></div>
+    <div class="col-md-10">
+      <div class="content-box-large">
+        <div class="panel-heading">
+          <div class="panel-title">
+            <h2><i class="glyphicon glyphicon-certificate"></i>&nbsp;&nbsp;Certificates</h2>
+          </div>
+        </div>
+        
+        <div class="exportSectionDiv">
+          <div style="float: left; width: 100%">
+            <h3><strong>Server Certificate</strong></h3>
+
+            <table class="table table-striped transactionTable">
+              <tbody>
+                <tr>
+                  <td width="120">Comman Name: </td>
+                  <td data-title="Comman Name">{{currentServerCert.commonName}}</td>
+                </tr>
+                <tr>
+                  <td>Organization</td>
+                  <td data-title="Organization">{{currentServerCert.organization}}</td>
+                </tr>
+                <tr>
+                  <td>Organization Unit</td>
+                  <td data-title="Organization Unit">{{currentServerCert.organizationUnit}}</td>
+                </tr>
+                <tr>
+                  <td>Country</td>
+                  <td data-title="Country">{{currentServerCert.country}}</td>
+                </tr>
+                <tr>
+                  <td>State</td>
+                  <td data-title="State">{{currentServerCert.state}}</td>
+                </tr>
+                <tr>
+                  <td>Locality</td>
+                  <td data-title="Locality">{{currentServerCert.locality}}</td>
+                </tr>
+                <tr>
+                  <td>Email Address</td>
+                  <td data-title="Email Address">{{currentServerCert.emailAddress}}</td>
+                </tr>
+                <tr>
+                  <td>Validity</td>
+                  <td data-title="Validity">
+                    Start: <strong>{{ currentServerCert.validity.start | date:'yyyy-MM-dd HH:mm:ss Z' }}</strong><br />
+                    End: <strong>{{ currentServerCert.validity.end | date:'yyyy-MM-dd HH:mm:ss Z' }}</strong></td>
+                </tr>
+                <tr>
+                  <td>Certificate</td>
+                  <td data-title="Certificate">{{currentServerCert.data}}</td>
+                </tr>
+              </tbody>
+            </table>
+
+            <div style="float: left; width: 48%">
+              <h4><strong>Upload Server Certificate</strong></h4>
+
+              <div ng-file-drop ng-file-select ng-model="serverCert" class="drop-box" style="padding: 10px" drag-over-class="dragover">Drop your certificate file here or click to upload</div>
+              <div ng-no-file-drop>File Drag/Drop is not supported for this browser</div>
+
+              <div ng-if="serverCert">
+                <h3>Uploaded Files: </h3>
```

Comment:
This bit looks a bit strange compared to the rest of the page. Perhaps once a file is uploaded we can just have a message below the drop area saying something like: "Newly uploaded file: <filename>"